chrome/content/zotero/containers/tagSelectorContainer.jsx

```diff
@@ -638,6 +638,49 @@ Zotero.TagSelector = class TagSelectorContainer extends React.PureComponent {
 		await Zotero.Tags.setColor(this.libraryID, io.name, io.color, io.position);
 	}
 
+	async openTagSplitterWindow() {
+		const oldTag = this.contextTag.name;
+		const dataIn = {
+			oldTag: this.contextTag.name,
+			isLongTag: false
+		};
+		const dataOut = { result: null };
+		
+		window.openDialog(
+			'chrome://zotero/content/longTagFixer.xhtml',
+			'',
+			'chrome,modal,centerscreen',
+			dataIn, dataOut
+		);
+
+		if (!dataOut.result) {
+			return;
+		}
+
+		const oldTagID = Zotero.Tags.getID(oldTag);
+
+		switch (dataOut.result.op) {
+			case 'split':
+				for (const library of Zotero.Libraries.getAll().filter(lib => lib.editable)) {
+					const itemIDs = await Zotero.Tags.getTagItems(library.id, oldTagID);
+					await Zotero.DB.executeTransaction(async () => {
+						for (const itemID of itemIDs) {
+							const item = await Zotero.Items.getAsync(itemID);
+							for (const tag of dataOut.result.tags) {
+								item.addTag(tag);
```

Comment:
This should use the old tag type as the second parameter, so that if you split automatic tags you still end up with automatic tags.